content/en/cosign/overview.md

```diff
@@ -15,98 +15,94 @@ features:
+
+```
+$ cosign verify-blob <file> --bundle cosign.bundle --certificate-identity=name@example.com 
+                              --certificate-oidc-issuer=https://accounts.example.com
+```
 
-### 3. Verify a container against a public key
+To verify, Cosign queries the transparency log (Rekor) to compare the public key bound to the certificate, and checks the timestamp on the signature against the artifact’s entry in the transparency log. The signature is valid if its timestamp falls within the small window of time that the key pair and certificate issued by the certificate authority were valid.
 
-This command returns `0` if *at least one* `cosign` formatted signature for the image is found
-matching the public key. Review the other sections of this site for information and caveats on other signature formats.
+### Working with containers
 
-Any valid payloads are printed to `stdout`, in JSON format.
-Note that these signed payloads include the digest of the container image, which is how we can be
-sure these "detached" signatures cover the correct image.
+Signing and verifying a container is similar to working with blobs.  The Cosign command to sign a container image is:
 
-```shell
-$ cosign verify --key cosign.pub dlorenc/demo
-The following checks were performed on these signatures:
-  - The cosign claims were validated
-  - The signatures were verified against the specified public key
-{"Critical":{"Identity":{"docker-reference":""},"Image":{"Docker-manifest-digest":"sha256:87ef60f558bad79beea6425a3b28989f01dd417164150ab3baab98dcbf04def8"},"Type":"cosign container image signature"},"Optional":null}
+```
+$ cosign sign <image URI>
 ```
 
-## Kubernetes Integrations
+This works the same as signing a blob, but the signature and certificate are attached as container metadata.
 
-Cosign comes with a few built-in Kubernetes integrations: `Secret` generation, and a [policy webhook](/policy-controller/overview/) `policy-controller`.
-In addition to the `policy-controller`, Cosign is also compatible with and supported by other policy engines such as:
+To verify a signed container image, use the following command:
 
-* [Conaisseur](https://github.com/sse-secure-systems/connaisseur#what-is-connaisseur)
-* [Kyverno](https://kyverno.io/docs/writing-policies/verify-images/)
-* [OPA Gatekeeper](https://github.com/sigstore/cosign-gatekeeper-provider)
+```
+$ cosign verify <image URI> --certificate-identity=name@example.com 
+                            --certificate-oidc-issuer=https://accounts.example.com
+```
+### Signing with a generated key
 
-To learn how to use Cosign with Kubernetes, review [Kubernetes](/cosign/kubernetes/).
+Although you do not have to use an existing key, you can generate a key or use an existing key for signing.  However, it is recommended that you use keyless signing, as a main feature of Sigstore is to make signatures invisible infrastructure.
```

Comment:
I understand what's meant here, but I think the first part of the sentence could be confusing to new users. Would something like this work?

"It it recommended that you use keyless signing, as a main feature of Sigstore is to make signatures invisible infrastructure that do not require key management. However, Sigstore allows you to use an existing key or generate a key if you prefer."

Reply:
OK, I changed the text.